### ISSUE TITLE
Notifications menu not accessible via Keyboard

### ISSUE THREAD

**Issue by davidbenque** [xwiki-labs/cryptpad#1201]
in the toolbar: ‘Open notifications panel’ under Notifications cannot be operated using the keyboard. The cause is that it is a  `<div>` that has been made clickable by means of JavaScript. 

This also causes issues for [criterion 4.1.2.](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
